app/controllers/chambers/documents_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Chambers::DocumentsController < ApplicationController
+  before_action :authenticate_user!
+  def create
+    document = Document.new(document_params)
```

Comment:
You might want to consider explicitly listing the parameters to Document.new, e.g.

```ruby
Document.new(candidate: params[:candidate], document: params[:document], ...)
```

Reply:
You also might want to then consider extracting the logic that finds a peer (in `document_params`) to a separate method.